### PR TITLE
Add additional process tree logging on harvester failure

### PR DESCRIPTION
**Pull request by anguss00**

### CODE REVIEW

File: AATAMS/AATAMS_sattag_dm/incoming_handler.sh [hunk 1]
```diff
@@ -100,7 +100,8 @@ handle_additions() {
     # finally, harvest the files!
     if ! index_files_bulk $UNZIPPED_DIR $AATAMS_SATTAG_DM_BASE $valid_new_mdb_files; then
         rm -f $valid_new_mdb_files
-        file_error "Failed indexing files, aborting operation..."
+        local process_tree=`pstree -aspG $$`
```
Comment (julian1) [inline AATAMS/AATAMS_sattag_dm/incoming_handler.sh]: We're going to end up as bash debugging experts.

Comment (anguss00) [inline AATAMS/AATAMS_sattag_dm/incoming_handler.sh]: The world leaders in bash debugging

**Comment by julian1** [comment]
My only (real) comment was whether `ps auxf` would be better, because it includes the arguments given to the process in the output, and it is normally part of a default install without needing apt-get.

But I can't see a way, to limit the output by specifying just the root process, rather than the tree of all processes.

**Comment by anguss00** [comment]
What about your bash call stack trick @julian1?

**Pull request merged**